test/specs/equipment.ts

```diff
@@ -0,0 +1,67 @@
+import { expect } from "@wdio/globals";
+import MainPage from "../pageobjects/main.page.ts";
+import UnitPage from "../pageobjects/unit.page.ts";
+import ProductsPage from "../pageobjects/products.page.ts";
+import * as testData from "../data/testdata.json";
+
+describe("Rentzila", () => {
+  it('C213- Checking ""Спецтехніка"" section on the main page', async () => {
+    await MainPage.open();
+
+    await expect(MainPage.equipmentTitle).toBeExisting();
+    await expect(MainPage.equipmentTitle).toHaveText("Спецтехніка");
+
+    await verifyEquipment(
+      MainPage.popularEquipmentLabel,
+      testData.equipment.popular.names,
+      testData.equipment.popular.namesInFilters,
+      testData.equipment.popular.namesInUnitPage
+    );
+    await verifyEquipment(
+      MainPage.agriculturalEquipmentLabel,
+      testData.equipment.agricultural.names,
+      testData.equipment.agricultural.namesInFilters,
+      testData.equipment.agricultural.namesInUnitPage
+    );
+    await verifyEquipment(
+      MainPage.buildingEquipmentLabel,
+      testData.equipment.building.names,
+      testData.equipment.building.namesInFilters,
+      testData.equipment.building.namesInUnitPage
+    );
+    await verifyEquipment(
+      MainPage.otherEquipmentLabel,
+      testData.equipment.other.names,
+      testData.equipment.other.namesInFilters,
+      testData.equipment.other.namesInUnitPage
+    );
+  });
+  async function verifyEquipment(
+    element: ChainablePromiseElement,
+    equipmentsNames: string[],
+    equipmentsNamesInFilter: string[],
+    equipmentsCategoryInProductsPage: string[]
+  ) {
+    await element.click();
+    var count: number = 7;
```

Comment:
@Anastas11aZakharova Hm, maybe here that is okey, but, what if for one of the tabs the quantity of the categories will change? Please, in order to avoit flakiness, check the actual length of the tabs (the requirements in the test case is not clear btw, should there be always the 7 sections for each tab every time or not - not specified, only for the 'Популярні' tab) and use it as a value for this parameter. Or, if it is required to check that there is always 7 sectios - please, add the assertion for that, at least, it will be cleare, where the script failed during the fixing process.

Reply:
Fixed in the latest commit
Added assertion that there always must be 7 sections